config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: controller
+  newTag: latest
```

Comment:
Can you explain this one?  It seems odd and out-of-order.

Reply:
Can we reorder it a bit more and put the `apiVersion` and `kind` at the top?  The rest of the reordering is fine.

Reply:
oh! yes. done.